mobile-app/app/screens/AppNavigator/screens/Auctions/AuctionNavigator.tsx

```diff
@@ -66,6 +80,32 @@ export function AuctionsNavigator (): JSX.Element {
           headerBackTestID: 'network_details_header_back'
         }}
       />
+
+      <AuctionsStack.Screen
+        component={PlaceBidScreen}
+        name='PlaceBidScreen'
+        options={{
+          headerTitle: () => (
+            <HeaderTitle
+              text={translate('screens/AuctionScreen', 'Place bid') + ' (Beta)'} // TODO: remove beta from title
```

Comment:
```suggestion
              text={translate('screens/AuctionScreen', 'Place Bid') + ' (Beta)'} // TODO: remove beta from title
```
for screen title we are using title case